packages/component/src/LiveRegion/private/LiveRegionSuggestedActions.tsx

```diff
@@ -30,5 +30,3 @@ LiveRegionSuggestedActions.propTypes = {
 };
 
 export default LiveRegionSuggestedActions;
-
-export type { LiveRegionSuggestedActions };
```

Comment:
This should be

```diff
- export type { LiveRegionSuggestedActions };
+ export type { LiveRegionSuggestedActionsProp };
```

However, we are not using the exported prop type. Thus, we can remove it.

In the future, we should use `PropsOf<T>` instead of exporting prop type.